src/lib/computeTraceOptionsFromSchema.js

```diff
@@ -0,0 +1,150 @@
+function computeTraceOptionsFromSchema(schema, _, context) {
+  // Filter out Polar "area" type as it is fairly broken and we want to present
+  // scatter with fill as an "area" chart type for convenience.
+  const traceTypes = Object.keys(schema.traces).filter(
```

Comment:
so, when is this code activated now?
When `advancedTraceTypeSelector` is not provided as a prop to the PlotlyEditor Component?
I think the user should still be able to configure the plot types even if they don't choose the `advancedTraceTypeSelector`.

So this should be adjusted and only read from schema if a trace config was not given.
And these should be the excluded types for now: 'area', 'violin', 'pointcloud', 'parcoords', 'sankey', 'carpet', 'scattercarpet', 'contourcarpet', 'scatterpolar'